src/new-client/src/OpenFin/DetachableTile/DetachableTile.tsx

```diff
@@ -27,16 +28,18 @@ interface Props {
   isTornOut: boolean
 }
 
-export const DetachableTile: React.FC<Props> = ({
-  symbol,
-  view,
-  isTornOut,
-}) => {
+export const DetachableTile: FC<Props> = ({ symbol, view, isTornOut }) => {
   const currencyPair = useCurrencyPair(symbol)
+  const ref = useRef<HTMLDivElement>(null)
 
   return (
-    <TearOutContext.Provider value={isTornOut}>
-      <Wrapper>
+    <TearOutContext.Provider
```

Comment:
would be good to pass down the ref via prop so that this context does not cause children re-render all the times due to it being an object now instead of a primitive boolean, not too fuss about it if it does not cause issues though